src/main/webapp/app/shared/notification/notification-popup/notification-popup.component.ts

```diff
@@ -40,14 +49,22 @@ export class NotificationPopupComponent implements OnInit {
      * @param notification {Notification}
      */
     navigateToTarget(notification: Notification): void {
-        this.router.navigateByUrl(this.notificationTargetRoute(notification));
+        // tslint:disable-next-line:triple-equals
+        if (notification.title == LIVE_EXAM_EXERCISE_UPDATE_NOTIFICATION_TITLE) {
+            const target = JSON.parse(notification.target!);
+            this.examExercisesNavigationService.navigateToExamExercise(target.exercise);
+        } else {
+            this.router.navigateByUrl(this.notificationTargetRoute(notification));
+        }
     }
 
     private notificationTargetRoute(notification: Notification): UrlTree | string {
         if (notification.target) {
             const target = JSON.parse(notification.target);
-
-            if (notification.title === 'Quiz started' && target.status) {
+            // tslint:disable-next-line:triple-equals
```

Comment:
why disable linter here?

Reply:
Look at my explanation above. (If you know a way to compare enums and strings with === please tell me)